scripts/dev/freeze.py

```diff
@@ -65,6 +65,7 @@ def get_build_exe_options(skip_html=False):
         ('qutebrowser/git-commit-id', 'git-commit-id'),
         ('qutebrowser/utils/testfile', 'utils/testfile'),
         ('qutebrowser/html', 'html'),
+        ('qutebrowser/3rdparty/pdfjs', '3rdparty/pdfjs'),
```

Comment:
This means freezing will fail when `update_3rdparty.py` wasn't run - that's probably good, but you should adjust `freeze.py` and `freeze_tests.py` accordingly.